plenum/server/view_change/view_changer.py

```diff
@@ -679,22 +625,6 @@ def _start_selection(self):
             self.provider.start_catchup()
             return
 
-        logger.info("{} starting selection".format(self))
-
-        node_reg = None
-        # in case of already completed view change
-        # use node registry actual for the moment when it happened
-        if self._is_propagated_view_change_completed:
-            assert self._accepted_view_change_done_message is not None
-            ledger_summary = self._accepted_view_change_done_message[1]
-            pool_ledger_size = ledger_summary[POOL_LEDGER_ID][1]
-            node_reg = self.provider.node_registry(pool_ledger_size)
-
-        if self.view_change_in_progress:
-            self.provider.restore_backup_replicas()
-
-        self.provider.select_primaries(node_reg)
```

Comment:
selection removed from view_changer

Reply:
Not sure that we should completely remove selection from here....

Reply:
Probably need to be discussed